README.md

```diff
@@ -409,7 +409,31 @@ intArrayPublisher.send([10, 2, 2, 4, 3, 8])
 ["10", "2", "2", "4", "3", "8"]
 ```
 
-------
+### TryMapMany
+
+Projects each element of a publisher collection into a new publisher collection form.
```

Comment:
I'm wondering if it's worth it in the long-term to have separate docs for try-prefixed vs non-throwing ... Perhaps worth just mentioning "You may also use tryMapMany if you wish to throw an error from your transformation" inside the regular tryMap docs? WDYT?